### PR TITLE
Comments for files associated with the Dashboard

### LINKED ISSUE #224: Comments for files associated with the Dashboard
Comments required in the following files:
- dashboardView.js
- CurrentTask.jsx
- CurrentTaskNotes.jsx
- CurrentTaskTimer.jsx
- TimerConfig.jsx
- TaskList.jsx

### PR DESCRIPTION
**Pull request by ireb15**
Comments added to the following files:
- dashboardView.js
- CurrentTask.jsx
- CurrentTaskNotes.jsx
- CurrentTaskTimer.jsx
- TimerConfig.jsx
- TaskList.jsx

### CODE REVIEW

**Review by xinyi98 — APPROVED** [review]
It looks good to me :)

File: src/components/current-task/CurrentTask.jsx [hunk 1]
```diff
@@ -16,6 +16,12 @@ import {
   FullscreenContext,
 } from "../timer-modal/TimerContextProvider";
 
+/**
+ * This function represents the Current Task component which shows
+ * a detailed view of a selected task from Today's Tasks, along
+ * with a timer that can be used to complete the current task.
+ * Timer configuration is availbale to the user.
```
Comment (brickahax) [inline src/components/current-task/CurrentTask.jsx]: Small typo 'available'

File: src/components/current-task/CurrentTaskTimer.jsx [hunk 2]
```diff
@@ -10,6 +10,11 @@ import {
 import IconButton from "../global/IconButton";
 import TimerModal from "../timer-modal/TimerModal";
 
+/**
+ * This function represents the Timer component of the Current Tasks component.
+ * The timer can be used when a user wishes to complete a task using the
+ * pomodoro technique. Timer conmfiguration is available to the user.
```
Comment (brickahax) [inline src/components/current-task/CurrentTaskTimer.jsx]: Small typo 'configuration'

File: src/components/current-task/CurrentTask.jsx (hunk 1 above)
Comment (ireb15) [reply src/components/current-task/CurrentTask.jsx]: Cheers

File: src/components/current-task/CurrentTaskTimer.jsx (hunk 2 above)
Comment (ireb15) [reply src/components/current-task/CurrentTaskTimer.jsx]: Cheers

**Review by brickahax — APPROVED** [review]
Good stuff bud 👌

**Review by thakurtushar02 — APPROVED** [review]

**Review by yuno99825 — APPROVED** [review]
Look's good, great job

**Pull request merged**